src/controllers/QueryRunner.ts

```diff
@@ -275,25 +275,18 @@ export default class QueryRunner {
                         if (self.shouldIncludeHeaders(includeHeaders)) {
                             let columnHeaders = self.getColumnHeaders(batchId, resultId, range);
                             if (columnHeaders !== undefined) {
-                                for (let header of columnHeaders) {
-                                    copyString += header + '\t';
-                                }
-                                copyString += '\r\n';
+                                copyString += columnHeaders.join('\t') + '\r\n';
```

Comment:
>\r\n [](start = 74, length = 4)

Minor: we should use os.EOL for the newline character instead or \r\n. On Unix (Mac and Linux) it's just \n. See [http://stackoverflow.com/questions/10864486/node-js-constant-for-platform-specific-new-line](http://stackoverflow.com/questions/10864486/node-js-constant-for-platform-specific-new-line)